index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1 id="title"> survey form for microverse </h1>
+<P id="description">This is a test for the trials</P> 
+
+<div>
+<form id="survey-form">
+
+  <label id="name-label">Name:</label><br>
+
+  <input type="text" id="name" required placeholder="Write your name"><br>
+
+  <label id="email-label">Email</label><br>
+
+  <input type="email" id="email" required placeholder="Write your email"><br><br>
+
+  <label id="number-label" >Age</label><br>
+
+  <input type="number" id="number" min="18" max="99" placeholder="write your age"><br><br>
```

Comment:
Use that many br tags make your code messy. Instead use CSS to make that kind of effects,

Reply:
♻️ Good suggestion there, using CSS to place elements on the page is probably better than using the `br` tags 🔥🔥. However, I would use a more polite way of using for changes. For example, rephrasing your comment to something like 
_Good job using the correct HTML forms while creating your form! However, please remember that laying out elements with CSS would be better than using multiple `br` tags. Kindly consider using CSS for that purpose_

You could also complement the requirement by mentioning briefly what can the dev use instead (flexbox, margins, padding, etc)